libparsec/crates/client/src/workspace/store/resolve_path.rs

```diff
@@ -23,10 +23,19 @@ use super::{
 /// that takes precedence.
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub(crate) enum PathConfinementPoint {
-    None,
+    NotConfined,
```

Comment:
Following our discussion about using that instead of Option<VlobID>, I'd rather use PathConfinementPoint anywhere it's possible (without to much hassle). 

But that will wait :hourglass: